tests/test_upload_commands.py

```diff
@@ -0,0 +1,200 @@
+"""
+Integration test for upload_commands.
+"""
+
+import dronekit
+
+from modules import upload_commands
+
+
+MAVLINK_FRAME = dronekit.mavutil.mavlink.MAV_FRAME_GLOBAL_RELATIVE_ALT
+MAVLINK_COMMAND = dronekit.mavutil.mavlink.MAV_CMD_NAV_WAYPOINT
+MAVLINK_TAKEOFF = dronekit.mavutil.mavlink.MAV_CMD_NAV_TAKEOFF
+MAVLINK_LANDING = dronekit.mavutil.mavlink.MAV_CMD_NAV_LAND
+DELAY = 3
+CONNECTION_ADDRESS = "tcp:localhost:14550"
+TOLERANCE = 0.00001

```

Comment:
Remove this in favour of using `math.isclose()` instead of `abs(a - b) <= TOLERANCE`